docs/blog/newsletter-april-2021.rst

```diff
@@ -0,0 +1,104 @@
+
+As for how anyone arrives at these reading time estimates, check out Nicholas Gryman's `reading-time <https://github.com/ngryman/reading-time>`_ project on GitHub. The Michael Marchese post also includes a basic formula that uses words per minute.
+
+----------------------
+Improving your editing
+----------------------
+
+Plenty of us do more rewriting than writing. Some are full or part-time editors; sometimes it makes sense for SMEs draft the content, and then a specialist writer fixes it up; some of us mostly work with existing material that needs restructuring or bringing up to date. So how do you improve your editing?
+
+Several documentarians agreed that the most important way to improve your editing is to edit more. Don't just read books or take training courses.
+
+That said, it's still interesting to read tips from great editors! Some recommendations: *On Writing Well* by Zinsser, *Bryson’s Dictionary for Writers and Editors* by Bill Bryson, *On Writing* by Stephen King; *The Elements of Eloquence* by Mark Forsyth; *The Sense of Style* by Steven Pinker; *The Oxford Style Manual* by OUP; *First You Write a Sentence* by Joe Moran. One documentarian got huge value from the Chartered Institue for Editing and Proofreading's resources and forums. 
+
+Another tip was to understand why the "rules" are there. Digging into linguistics and anthropology can help you figure out if a rule is actually helpful or not. And a grammar reference like *The Gregg Reference Manual* can be helpful day-to-day.
+
+Finally, editing challenges are sometimes not about words but relationships. Some authors have very set ideas about writing! So communication and conflict resolution training can be useful. If you work repeatedly with the same people, it's worth learning their quirks, over time giving feedback on some common problems they could avoid. Style guides can also help lower the temperature of a disgreement: sometimes it's productive to explain your reasoning, but other times appeals to authority can save a lot of energy!
+
+----------------------------------------
+What we're reading/listening to/watching
+----------------------------------------
+
+The #bipoc group's been reading the following materials on diversity, inclusion, and equity. Want to join the conversation? Join us in the #bipoc slack channel!
+
+**Small**: `Words Matter: Finally, Tech Looks at Removing Exclusionary Language <https://thenewstack.io/words-matter-finally-tech-looks-at-removing-exclusionary-language>`__ by Jennifer Riggins, June 2020. It discusses the tech industry’s look at the language it uses, because words do matter.
```

Comment:
Small / medium / large feels odd. If we're gonna use them can we specify what we're talking about? Time investment?

Reply:
Yes - sorry I should have explained. The concept here was to give folks an option on what level they wanted to consume this heavy information. The first is a short article, the second is a 20+ minute podcast, and the last is a very long hour+ minute video.

Reply:
Maybe something conversational, like this?

For a short read, check out...
If you have a little more time, listen to...
If you can invest a bit more than 1 hour, watch...

Reply:
I like Hilary's suggestions so I'll use those for now - I don't mind repeating them even if they're a little less generic.